lnst/Tests/LongLivedConnections.py

```diff
@@ -0,0 +1,167 @@
+import time
+import select
+import socket
+import logging
+import resource
+import threading
+from typing import Union
+
+from .BaseTestModule import BaseTestModule
+from lnst.Common.IpAddress import Ip4Address, Ip6Address, ipaddress
+from lnst.Common.Parameters import IntParam, StrParam
+
+
+def get_af(ip: Union[Ip4Address, Ip6Address]):
```

Comment:
ok so... this should have been asked earlier but i somehow missed it...

why have this function? the `IpAddress` class already defines a `family` attribute which returns the correct value for ipv4 and ipv6: https://github.com/LNST-project/lnst/blob/master/lnst/Common/IpAddress.py#L31

i assume that this is because `server_ip` and `client_ip` are `StrParam` so you want to do the conversion... but... why are they `StrParam`? shouldn't they just be `IpParam`? https://github.com/LNST-project/lnst/blob/master/lnst/Common/Parameters.py#L91